### PR TITLE
Al/BE-95 Functions for accounts pallet.

### PR DESCRIPTION
**Pull request by lut1k**
Implemented fucntion is_admin and unit-test for it in accounts pallet.
 moved MAX_MEMBERS and INITIAL_EXCHANGE_RATE constants to constants.rs file.
Implemented is_admin_internal function and unit test for it in accounts pallet.

### CODE REVIEW

**Review by Pavel-Blaize — APPROVED** [review]

File: pallets/accounts/src/lib.rs [hunk 1]
```diff
@@ -29,8 +33,12 @@ decl_event!(
 	{
 		/// New account is added to the allow-list: \[who\]
 		AccountAdded(AccountId),
+
 		/// Account is removed from the allow-list: \[who\]
 		AccountRemoved(AccountId),
+
+		/// The caller is a member: \[who\]
+		IsAMember(AccountId),
```
Comment (Apollo50) [inline pallets/accounts/src/lib.rs:41]: Perhabs better to rename this to IsAnAdmin?

File: pallets/accounts/src/lib.rs [hunk 2]
```diff
@@ -39,8 +47,8 @@ decl_error! {
 		/// The account cannot be added to the allowed list because it has already been added.
 		AlreadyMember,
 
-		/// The account cannot be removed from the allowed list because it is not a member
-		NotMember,
+		/// The account cannot be removed from the allowed list because it is not a member.
+		NotAMember,
```
Comment (Apollo50) [inline pallets/accounts/src/lib.rs:51]: the same

**Review by Apollo50 — DISMISSED** [review]

File: pallets/accounts/src/lib.rs (hunk 1 above)
Comment (lut1k) [reply pallets/accounts/src/lib.rs:41]: fixed

File: pallets/accounts/src/lib.rs (hunk 2 above)
Comment (lut1k) [reply pallets/accounts/src/lib.rs:51]: > the same

fixed

**Pull request merged**